Packs/Arcanna/Integrations/ArcannaAI/ArcannaAI.py

```diff
@@ -0,0 +1,243 @@
+import demistomock as demisto  # noqa: F401
+from CommonServerPython import *  # noqa: F401
+from CommonServerUserPython import *  # noqa: F401
+
+import json
+import urllib3
+import traceback
+import requests
+from typing import Any, Dict
+
+# Disable insecure warnings
+urllib3.disable_warnings()
+
+''' CONSTANTS '''
+
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
+MAX_INCIDENTS_TO_FETCH = 50
+SEVERITES = ['Low', 'Medium', 'High', 'Critical']
+
+''' CLIENT CLASS '''
+
+
+class Client:
+    """ Implements Arcanna API
+    """
+
+    def __init__(self, verify=True, proxy=False, api_key=None, base_url=None, default_job_id=-1):
+        self.base_url = base_url
+        self.verify = verify
+        self.proxy = proxy
+        self.api_key = api_key
+        self.default_job_id = default_job_id
+
+    def get_headers(self):
+        """   Adds header
+
+        """
+        headers = {
+            'accept': 'application/json',
+            'x-arcanna-api-key': self.api_key
+        }
+        return headers
+
+    def set_default_job_id(self, job_id):
+        self.default_job_id = job_id
+
+    def get_default_job_id(self):
+        return self.default_job_id
+
+    def test_arcanna(self):
+        url_suffix = 'api/v1/health'
+        raw_response = requests.get(url=self.base_url + url_suffix, headers=self.get_headers(), verify=self.verify)
+        return json.loads(raw_response.text)
+
+    def list_jobs(self):
+        url_suffix = 'api/v1/jobs'
+        raw_response = requests.get(url=self.base_url + url_suffix, headers=self.get_headers(), verify=self.verify)
+        if raw_response.status_code != 200:
+            raise Exception("Error HttpCode. Forbidden")
+        return json.loads(raw_response.text)
+
+    def send_raw_event(self, job_id=None, severity=None, title=None, raw_body=None):
```

Comment:
I don't get this suggestion .it does not  make sense

Reply:
the `job_id, title, and raw_body` arguments are required for the `send_raw_event` function execution. It is not a best practice to initiate these values with `None` because in that case, the request would fail on bad/missing data.

It is not a mandatory change but only a suggestion.